### PR TITLE
Typography updates

### PR DESCRIPTION
**Pull request by alexmanzo**
## Changes Made
- Update font family to use Montserrat and Open Sans
- Update link styles
- Reformat "welcome" blurb on homepage

## Notes
- My code formatter might have gone a little rogue in a couple spots - let me know if something got changed incorrectly.
- Let me know what you think on the homepage format. I tried a couple things and this seemed to work best.
 - I updated the footer with these link styles, although would like to update it to the [new format](https://docs.google.com/presentation/d/1R1DBlggMBDN-Ct5q2494mEY7mq1ksJjw8JGA57h91bc/edit#slide=id.g50c3f3d9bd_3_96), which should hopefully make it look a little better.
- The link underlines are actually set up a pseudo-elements. It makes links that break on two lines ([example](http://localhost:8080/en/compass/PTGNRL/blockgroup) under Additional Resources) lose the underline across every word. we discussed re-formatting that area anyway, so I would hope to address it there.
- The other problem spot I noticed is a link for Duke's Center for Population Health Improvement - under Additional resources (link)[http://localhost:8080/en/compass/CKD_ASIAN/tract?selected=370630020183&selected=370630020201]. This isn't adding pseudo-element as expected. Strangely, when I select it in Inspector, it shows this whole block as selected. 
Screen Shot 2019-10-30 at 6 42 31 PM

### CODE REVIEW

**Comment by timstallmann** [comment]
Overall this looks great, thx so much! The cleaner type really does impact the look site-wide in a positive way. 

A few comments:
* I think the link underline not wrapping enough of a weird look that we'll want to address it overall. It seems like there's a a [javascript fix for this](https://jsfiddle.net/6zm8L9jq/1/) which may not work as-is in Vue but I wonder if since all the links are in an `<ExternalLink>` element if there's some way to add an `mounted()` hook to that component which implements a fix for the link underline wrap. Do you want to take a crack at that? Or I can work on it too if not..

* For some reason the bottom corners of the images on front page aren't getting the border-radius that the top corners are, this is on FF
Screen Shot 2019-10-31 at 11 51 37 AM

* Do you have a sense of which weights of Montserrat & Open Sans we're using on the site? If it's possible to narrow down the weight selector on the google CSS URL that could save some time on the initial page load...

File: src/js/components/dashboard-footer.vue [hunk 1]
```diff
@@ -56,21 +56,42 @@
 </template>
 
 <script>
-import ExternalLink from './external-link';
+import ExternalLink from "./external-link";
```
Comment (timstallmann) [inline src/js/components/dashboard-footer.vue]: this is what you were talking about with the code formatter? i'm fine leaving it for now and I have a note to implement some sort of more consistent code formatting specs on the project anyway..

File: src/scss/typography.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
```
Comment (timstallmann) [inline src/scss/typography.scss]: Is there a reason these two lines are also in `main.scss` or is that a typo?

File: src/js/components/dashboard-footer.vue (hunk 1 above)
Comment (alexmanzo) [reply src/js/components/dashboard-footer.vue]: Yep - sounds good!

File: src/scss/typography.scss (hunk 2 above)
Comment (alexmanzo) [reply src/scss/typography.scss]: Typo, will remove!

**Comment by alexmanzo** [comment]
Hi @timstallmann everything should be addressed with this update. The reason I didn't update the `External Link` component specifically is because any links in the metadata aren't generated from that component, so weren't getting the same markup.

This worked well for me locally, let me know if you run into any issues!

**Comment by timstallmann** [comment]
@alexmanzo this is looking great!! I went ahead and pushed minor fixes to a few bugs and was looking through our notes and realized there were a few tweaks that I needed to do on the typography & homepage, so went ahead and pushed those too.

Now, the only problem is that links inside the 1,2,3 step blue boxes don't show up very well -- do you have any sense what to do easily about that? Otherwise, I'm good to merge.

**Comment by alexmanzo** [comment]
Hi @timstallmann, I made some styling updates to treat those a little differently.

**Comment by timstallmann** [comment]
Great, thanks! I realized I made a bug when I was trying to set the width of the container only for the report, which I'll fix tmrw and merge this. Showed off to DW staff this morning and everyone was really impressed at your work!

**Comment by alexmanzo** [comment]
Sounds good! I think I have that solved in a non-buggy way on my other branch. I'll follow-up with a PR tomorrow or this weekend on report stuff.

**Pull request merged**